enrollment-server-onboarding/pom.xml

```diff
@@ -131,6 +131,19 @@
             <artifactId>logstash-logback-encoder</artifactId>
         </dependency>
 
+        <!-- Standalone run -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-tomcat</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.tomcat.embed</groupId>
+            <artifactId>tomcat-embed-el</artifactId>
```

Comment:
This is a little bit more tricky. It is already a part of `spring-boot-starter-tomcat` but also a part of `spring-boot-starter-validation`, see the result of `mvn dependency:tree`

```
[INFO] +- org.springframework.boot:spring-boot-starter-validation:jar:3.1.4
[INFO] |  +- org.apache.tomcat.embed:tomcat-embed-el:jar:10.1.13
```
That's why the scope is defined `test` in the parent pom. Now it is possible to change it to `provided` and remove the declaration here. Maven always check that via enforcer plugin, you may manually run it as `mvn validate`.